Default limit of 100 is too small for some things (like sci-chemistry/gamess)

Patch by Markus Dittrich

http://bugs.gentoo.org/114367

--- gcc-3.4.4/libf2c/libI77/fio.h
+++ gcc-3.4.4/libf2c/libI77/fio.h
@@ -86,7 +86,7 @@
 #define errfl(f,m,s) do {return err__fl((int)f,m,s);} while(0)
 
 /*Table sizes*/
-#define MXUNIT 100
+#define MXUNIT 512
 
 extern int f__recpos;		/*position in current record */
 extern int f__cursor;		/* offset to move to */
